test/appium/tests/atomic/chats/test_public.py

```diff
@@ -126,3 +126,35 @@ def test_send_korean_characters(self):
         if not public_chat.chat_element_by_text(message).is_element_displayed():
             self.errors.append('Message with korean characters is not shown')
         self.verify_no_errors()
+
+    @marks.testrail_id(5336)
+    @marks.high
+    def test_user_can_interact_with_public_chat(self):
+        signin = SignInView(self.driver)
+        home_view = signin.create_user()
+        chat = home_view.join_public_chat('status')
```

Comment:
we definitely should not use #status public chat in the test

Reply:
> we definitely should not use #status public chat in the test
reasonable. just took it from test case steps.